swift/ChartboostMediationDemo/Ads/Banner/BannerAdViewController.swift

```diff
@@ -20,22 +20,18 @@ import UIKit
 ///
 class BannerAdViewController: UIViewController {
 
-    /// The banner's container view. In the storyboard, it is constrained to be 320x50 for the sake of this
-    /// demo, which only uses the fixed `standard` size.
-    @IBOutlet private var bannerContainer: UIView!
```

Comment:
I opted to remove the container and just add the banner to the view controller's view, which I feel better demonstrates adaptive banners.